features/run/service_endpoint_environment_variables.feature

```diff
@@ -1,12 +1,14 @@
-Feature: Service endpoint enviornment variable
+Feature: Service endpoint enviornment variables
 
   As an Exosphere developer
-  I want the services to know where each other are available from the outside world
-  So clients can talk directly to other services
+  I want the services to know where each other are available
+  So that communication between services are possible
 
   Rules:
   - "<NAME>_EXTERNAL_ORIGIN" is passed as an environment variable to
     each service for any public service
+  - "<NAME>_ORIGIN" is passed as an environment variable to
```

Comment:
thoughts on making this env var `<NAME>_INTERNAL_ORIGIN`. I think that is more clear.